CHANGELOG.md

```diff
@@ -760,7 +760,7 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
-[develop]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.31.0...HEAD
```

Comment:
This needs to be updated, not dropped 😉

Reply:
This PR got created by the release cli. I think when we finish the release, there will be another PR that will add the release date and another entry here, from `v3.32.0...HEAD`

Reply:
I'm pretty sure we do this manually all the time 🤣 Maybe we don't have to but just didn't know.

Reply:
Yeah, I also did this manually all the time before. But this time created automatically.